modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -241,8 +243,7 @@ SMB storage. ownCloud supports SMB notifications with an `occ` command,
 
 `occ wnd:listen`.
 
-NOTE: The notifier only works with remote storage on Windows servers. It does not work reliably with Linux
-servers due to technical limitations.
+NOTE: The listener capabilities depend on the ability of the used SMB/CIFS storage to provide notifications. Some of those, will have restrictions. Please check with your respective storage provider. Notifications for Samba only works for the target folder you're listening, but they won't go deep down. If you're listening on the "/top" folder, you won't receive notifications for "/top/middle/bottom" folder. You'd need to setup listeners for every existing folder and also for the new ones that could be created. Windows doesn't have this limitation. You'll receive notifications for every file or folder inside the folder you're listening to.
```

Comment:
```suggestion
NOTE: The capabilities of the listener depends on the ability of the used SMB/CIFS storage backend to provide notifications. While Windows file servers have no limitations, some vendors may have restrictions. Please check these with your storage provider. It may be possible, that notifications for Samba only work for the target folder you're listening to, but not for any sub structures. If you're listening on the "/top" folder, you may not receive notifications for "/top/middle/bottom" folder. In this case, you have to setup listeners for every _existing_ folder and also for any _new_ folders that will be created. Windows file servers do not have this limitation. You'll receive notifications for every file or subfolder inside the folder you're listening to.
```

Reply:
The capabilities is plural. Plural does not need the "s" behind the verb.

Sounds like you really want people to use windows since you have it twice now in the document. Once in the top "While Windows has no limitation" and again at the bottom "Windows servers have no limitation" Seems redundant to me

Reply:
I changed it, can you re-look?

Reply:
on further review - this is too big for a "note" 

A note should be like "Watch out, there might be dragons" but not 9 lines of text.

I think this text should go in to the limitations part that has to be added.

What do you think?

Reply:
This is a good idea!
You may commit that text and rebase your local env. Then take that text and move it into "limitations" and add a note with text to look at limitations.